corehq/apps/sms/views.py

```diff
@@ -951,17 +952,17 @@ def _fmt_backend_data(self, backend):
         is_editable = not backend.is_global and backend.domain == self.domain
         if len(backend.supported_countries) > 0:
             if backend.supported_countries[0] == '*':
-                supported_countries = _('Multiple%s') % '*'
+                supported_countrie_names = _('Multiple%s') % '*'
             else:
-                supported_countries = ', '.join(
-                    [_(c) for c in backend.supported_countries])
+                supported_countrie_names = ', '.join(
+                    [_(country_name_from_code(int(c))) for c in backend.supported_countries])
         else:
-            supported_countries = ''
+            supported_countrie_names = ''
```

Comment:
country is misspelled here - I'm not going to block on it but hope you will correct it